core/src/main/java/org/ehcache/core/internal/store/StoreConfigurationImpl.java

```diff
@@ -37,21 +37,21 @@
   private final ResourcePools resourcePools;
   private final Serializer<K> keySerializer;
   private final Serializer<V> valueSerializer;
-  private final int orderedEventParallelism;
+  private final int dispatcherConcurrency;
 
   /**
    * Creates a new {@code StoreConfigurationImpl} based on the provided parameters.
    *
    * @param cacheConfig the cache configuration
-   * @param orderedEventParallelism the level of parallelism for ordered events
+   * @param dispatcherConcurrency the level of parallelism for ordered events
```

Comment:
Should we use concurrency instead of parallelism in doc also. just for being consistent